assembl/static/js/app/views/tokenVoteSession.js

```diff
@@ -1464,6 +1482,20 @@ var TokenVoteSessionModal = Backbone.Modal.extend({
     
     
     Promise.join(collectionManager.getUserLanguagePreferencesPromise(Ctx), collectionManager.getAllIdeasCollectionPromise(), function(userLanguagePreferences, allIdeasCollection) {
+      that.userLanguagePreferences = userLanguagePreferences;
+      var settings = that.widgetModel.get("settings") || {};
+      var items = "items" in settings ? settings.items : [];
+      var question_item = items.length ? items[0] : null;
+
+      var questionTitleLangString = new LangString.Model();
+      questionTitleLangString.initFromObjectProperty(question_item, 'question_title');
+      that.$('.question-title').text(questionTitleLangString.bestValue(that.userLanguagePreferences));
```

Comment:
I would add more conditions around this (or move this logic into `bestValue`, if there is no `userLanguagePreference`, then `firstOriginal` is returned. Because, currently, if there is no langPref, the method throws an error, so this crash would become very visible.

Perhaps you could get the `bestValue`'ed lang as a var, and if it doesn't exist, return the original in the `$(..).text()`.

Reply:
If a user did not set any langpref a default one is received I think. Ensuring that getUserLanguagePreferencesPromise contains a valid langpref is probably the job of this getUserLanguagePreferencesPromise function?

Reply:
No, the `getUserLanguagePreferencesPromise` doesn't return a default. I don't think the API is returning a default lang_pref from configs, btw. So in rare case (now) that user doesn't have lang_pref, returns empty collection.

Reply:
ok I think I solved this potential issue in 85dad6b06f0f81bf41dea9d28b36715b47ae7cf0 and 19a7201594254bf3905c993ab7cffe2e4f1e44ef. But for a better solution, I think a full reevaluation/clarification on how langprefs should be used in combination with langstrings is necessary